registries/adapters/apiv2_adapter.go

```diff
@@ -75,6 +80,35 @@ func NewPartnerRhccAdapter(config Configuration) (PartnerRhccAdapter, error) {
 	return PartnerRhccAdapter{apiV2}, nil
 }
 
+func NewRegistryProxyAdapter(config Configuration) (RegistryProxyAdapter, error) {
+	// we want to use a different token for this adapter, so do not call
+	// NewAPIV2Adapter directly
+	apiv2a := APIV2Adapter{
+		config: config,
+		client: oauth.NewClient(config.User, config.Pass, config.SkipVerifyTLS, config.URL),
+	}
+
+	log.Debugf("XXX number of images configured: %d", len(config.Images))
```

Comment:
remove this, or remove the XXX at least?

Reply:
removed and forced push to keep this commit clean :)